### ISSUE TITLE
Unit and functional tests

### ISSUE THREAD

**Issue by adamkiczula** [yuriy1337/Lunch-Randomizer#33]
We need to get our unit and functional tests in and make sure we keep them up to date, then we will know next time Tom breaks the build...

**Comment by tomarra** [comment]
Agreed. I like breaking the build to keep the other developers on their toes :P

**Comment by yuriy1337** [comment]
Whoever breaks the build buys everyone donuts. By saying that, Ill probably the first to break it.
